src/features/auth/hooks/validate-username-email.ts

```diff
@@ -8,26 +8,42 @@ const USERNAME_BLACKLIST = ['root'];
 
 for (const method of ['POST', 'PATCH'] as const) {
 	hooks.addPureHook(method, 'resin', 'user', {
-		POSTPARSE: async ({ request, tx }) => {
+		POSTPARSE: async ({ request }) => {
 			if (request.values.username != null) {
 				request.values.username = normalizeHandle(request.values.username);
 				if (USERNAME_BLACKLIST.includes(request.values.username)) {
 					throw new ConflictError('This username is blacklisted');
 				}
-
-				const existingUser = await findUser(request.values.username, tx, [
-					'id',
-				]);
-				if (existingUser) {
-					throw new ConflictError('This username is already taken');
-				}
-			}
-			if (request.values.email != null) {
-				const existingUser = await findUser(request.values.email, tx, ['id']);
-				if (existingUser) {
-					throw new ConflictError('This email is already taken');
-				}
 			}
 		},
 	});
+
+	hooks.addPureHook(method, 'resin', 'user', {
```

Comment:
```suggestion
	});

	hooks.addPureHook(method, 'resin', 'user', {
```

Reply:
I guess you mean to combine the two right? B/c the code suggestion doesn't seem to have any change :thinking: